src/marten/db/query/set.cr

```diff
@@ -754,6 +830,86 @@ module Marten
           raise Errors::MultipleRecordsFound.new("Multiple records (#{results.size}) found for get query")
         end
 
+        # Returns the model instance matching the given raw SQL condition, raising an error if not found.
+        #
+        # This method allows retrieving a record based on a custom SQL condition without parameters.
+        # If no record matches the condition, a `RecordNotFound` exception is raised.
+        #
+        # Example:
+        # ```
+        # tag = Tag.all.get!("is_active = true")
+        # ```
+        def get!(raw_predicate : String)
+          raise_empty_raw_predicate if raw_predicate.empty?
+          get!(Node.new(raw_predicate))
+        end
+
+        # Returns the model instance matching the given raw SQL condition with positional arguments, raising an
+        # error if not found.
+        #
+        # This method allows retrieving a record based on a custom SQL condition with positional arguments.
+        # If no record matches the condition, a `RecordNotFound` exception is raised.
+        #
+        # Example:
+        # ```
+        # tag = Tag.all.get!("name=?", "crystal")
+        # ```
+        def get!(raw_predicate : String, *args)
+          get!(raw_predicate, args.to_a)
+        end
```

Comment:
I am only seeing specs for the named parameters variation of the `#get!` method when used with raw predicates. Can we ensure this one is unit tested too? 🙏